ui/apps/platform/src/Containers/VulnMgmt/List/Cves/VulnMgmtListCves.js

```diff
@@ -246,6 +246,17 @@ export function getCveTableColumns(workflowState) {
         },
     ];
 
+    if (currentEntityType === entityTypes.NODE_CVE || currentEntityType === entityTypes.IMAGE_CVE) {
+        tableColumns.splice(3, 0, {
```

Comment:
I'd recommend including this column within the original list with a ternary : null, and let it get filtered out if not one of those two types, but still will work, since it is a short-lived solution.